src/wp-admin/menu.php

```diff
@@ -241,23 +241,23 @@
 
 unset( $appearance_cap );
 
-// Add 'Theme Editor' to the bottom of the Appearance menu.
+// Add 'Theme File Editor' to the bottom of the Appearance menu.
 if ( ! is_multisite() ) {
 	// Must use API on the admin_menu hook, direct modification is only possible on/before the _admin_menu hook.
 	add_action( 'admin_menu', '_add_themes_utility_last', 101 );
 }
 /**
- * Adds the 'Theme Editor' link to the bottom of the Appearance or Tools menu.
+ * Adds the 'Theme File Editor' link to the bottom of the Appearance menu.
  *
  * @access private
  * @since 3.0.0
- * @since 5.9.0 'Theme Editor' link has moved to the Tools menu when a block theme is active.
+ * @since 5.9.0 'Theme Editor' link was renamed to 'Theme File Editor'.
  */
 function _add_themes_utility_last() {
 	add_submenu_page(
-		wp_is_block_theme() ? 'tools.php' : 'themes.php',
-		__( 'Theme Editor' ),
-		__( 'Theme Editor' ),
+		'themes.php',
```

Comment:
@audrasjb See my comment/thinking here https://github.com/WordPress/wordpress-develop/pull/1948#issuecomment-998943854. I now agree that the Theme File Editor is a tool that belongs under "Tools".

```suggestion
		wp_is_block_theme() ? 'tools.php' : 'themes.php',
```